pkg/steps/template.go

```diff
@@ -553,7 +553,7 @@ func waitForPodCompletionOrTimeout(ctx context.Context, podClient coreclientset.
 				log.Printf("warning: failed to get pod %s: %v", name, err)
 				continue
 			}
-			if pod.Status.Phase != coreapi.PodRunning && time.Since(pod.CreationTimestamp.Time) > 30*time.Minute {
+			if !podHasStarted(pod) && time.Since(pod.CreationTimestamp.Time) > 30*time.Minute {
```

Comment:
Can we reduce the time? We picked that big timeframe explicitly because it includes `initContainers` but now we would assume a couple of mins (10 tops?) to be sufficient, right? The worst case is basically `We have to create a new node and download our image there`

Reply:
Updated.  I also changed the implementation slightly after I realized the check would become useless for multi-stage tests, which always have one init container.